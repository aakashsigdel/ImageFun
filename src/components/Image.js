import React from 'react';

import './Image.scss';

const Image = props =>
  <img src={props.imageURL} className={props.actions}/>;

export default Image;
