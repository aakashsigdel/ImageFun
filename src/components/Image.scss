.rotate {
  transform: rotate(75deg);
}

.translateX {
}

.opacity {
  opacity: 0.5;
}

.bounce {
}

.reset {
}

