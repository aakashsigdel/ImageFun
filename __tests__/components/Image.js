import React from 'react';
import Image from '../../src/components/Image';

import renderer from 'react-test-renderer';

describe('<Image /> component', () => {
  it('should render <Image /> correctly', () => {
    const tree = renderer.create(<Image />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});

