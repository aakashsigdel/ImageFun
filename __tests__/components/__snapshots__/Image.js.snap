exports[`<Image /> component should render <Image /> correctly 1`] = `
<img
  className={undefined}
  src={undefined} />
`;
